extension/data/deinflect.dat

```diff
@@ -142,6 +142,8 @@ past negative
 きます	くる	2176	13
 ぎます	ぐ	640	13
 くない	い	1028	15
+です		1028	13
```

Comment:
1028 is probably not the right one for these two since it means i-adj -> i-adj and we want one which is OTHER -> i-adj and that looks like 1152 based on other examples. 

```suggestion
です		1152	13
```